bindings/go/evmc/evmc.go

```diff
@@ -42,7 +42,7 @@ static struct evmc_result execute_wrapper(struct evmc_vm* vm,
 		input_data,
 		input_size,
 		*value,
-		*create2_salt,
+		{{0}}, // create2_salt: not required for execution
```

Comment:
I changed that from the original commit because `= {}` is not valid ISO C syntax.